webgpu.h

```diff
@@ -901,6 +923,15 @@ WGPU_EXPORT void wgpuRenderPassEncoderSetViewport(WGPURenderPassEncoder renderPa
 WGPU_EXPORT WGPUTextureView wgpuTextureCreateView(WGPUTexture texture, WGPUTextureViewDescriptor const * descriptor);
 WGPU_EXPORT void wgpuTextureDestroy(WGPUTexture texture);
 
+// Methods of SwapChain
+WGPU_EXPORT WGPUTextureView wgpuSwapChainGetCurrentTextureView(WGPUSwapChain swapChain);
+WGPU_EXPORT void wgpuSwapChainPresent(WGPUSwapChain swapChain);
+
+// Surface creation
+WGPUSurface wgpuCreateSurfaceFromMetalLayer(void *layer);
+WGPUSurface wgpuCreateSurfaceFromWindowsHwnd(void *_hinstance, void *hwnd);
```

Comment:
We should be able to remove the `_` prefix.

It was just added there temporarily because we don't use `hinstance` internally in wgpu yet :) (the prefix avoids an unused parameter warning in Rust)